journeys/et/sessionRestore.ts

```diff
@@ -1,16 +1,16 @@
-import { prompt } from "inquirer"
-import { findPreviousSessions, restorePreviousSession } from "app/session"
-import { Journey } from "types/journey"
-import { getIdealSizeForInquirer } from "app/helpers"
-import { loadCurrentSessionIntoMemory } from "app/et/configs"
+import { prompt } from 'inquirer'
+import { findPreviousSessions, restorePreviousSession } from 'app/session'
+import { Journey } from 'types/journey'
+import { getIdealSizeForInquirer } from 'app/helpers'
+import { loadCurrentSessionIntoMemory } from 'app/et/configs'
 
-const QUESTION_PREVIOUS_SESSION = "Select a previous session"
+const QUESTION_PREVIOUS_SESSION = 'Select a previous session'
 
 async function restoreSession() {
   const prevSessions = await findPreviousSessions()
 
   if (!prevSessions.length) {
-    console.warn(`There are no previous sessions found`)
```

Comment:
Why warn and not stderr?

Reply:
Its not really an error if there are no previous sessions. But a warning in case the user expected there to be some